spec/app/github_repo_fetcher_spec.rb

```diff
@@ -12,17 +12,21 @@
 
   describe "#repo" do
     it "fetches a repo from cache if it exists" do
-      allow(CACHE).to receive(:fetch).with("all-repos", hash_including(:expires_in)) do
+      cache = double("CACHE")
```

Comment:
The use of CACHE in this class seems like a smell, given that it's a Singleton - it shouldn't need to use a global constant to cache things. However, I see that CACHE gets used in quite a few places, so not really in scope here to change that.

I do think there's a small risk of this code regressing, if someone thinks the `allow` form is more concise in future. Perhaps we could DRY it up and preempt that by using a helper method in here?

Reply:
> I was a bit surprised and confused by the second commit, as it seems to be unrelated to the PR.

Yes, totally understand that. I opened this PR for the relative links fix, then found the build failed, so tacked on a fix commit that had been on my to-do list for a while 😅 thanks for reviewing it.

Reply:
I've just pushed up a commit to tidy this up 👍

Reply:
Thanks, that's much simpler - feel free to merge.